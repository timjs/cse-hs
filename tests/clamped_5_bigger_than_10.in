5
plus(b,b(pow,mult))
minus(minus,b(x,b))
f(pow(plus,f),plus)
mult(x,b(pow,mult))
minus(minus(x,x),b)
