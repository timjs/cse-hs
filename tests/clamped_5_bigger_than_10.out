plus(b,b(pow,mult))
minus(minus,b(x,b))
f(pow(plus,f),3)
mult(x,b(pow,mult))
minus(minus(x,3),b)
